frontend/web/src/app/stl-viewer/Viewer.ts

```diff
@@ -815,8 +839,9 @@ export class Viewer {
         if (!this.isLoaded) {
             return;
         }
-
         if (e.touches.length > 0) {
+            console.log(typeof e.touches.length)
+            this.setMouseUsage(e.touches.length);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') setMouseUsage is deprecated: This method is obsolete since version 1.5.0 and may be removed in the future. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS1874)